css/templates/travel/page-vars.css

```diff
@@ -112,3 +137,95 @@
   @custom-media --breakpoint-md-lg (min-width: 52.06rem) and (max-width: 72rem);
   @custom-media --breakpoint-v-md-lg (min-height: 52.06rem) and (max-height: 72rem);
 }
+
+body {
+  color: var(--font-color-primary);
+  font-family: var(--font-family-primary);
+  line-height: var(--line-height-2);
+  box-sizing: border-box;
+  margin: 0;
+  -webkit-font-smoothing: antialiased;
+}
+
+*, *::before, *::after {
+  box-sizing: inherit;
+}
+
+h1, h2, h3, h4, h5, h6, p {
+  margin: 0;
+}
+
+a {
```

Comment:
we can use the text-decoration-none bass class where needed. Dropping the underline on all the links on a site as the default is a bad idea

Reply:
Agreed